src/Provider/Geonames/Geonames.php

```diff
@@ -92,6 +98,73 @@ public function reverseQuery(ReverseQuery $query): Collection
     }
 
     /**
+     * @param string|null $country
+     * @param string|null $locale
+     * @param int|null    $maxRows
+     * @param int|null    $startRow
+     * @return AddressCollection
+     * @throws \Geocoder\Exception\Exception
+     */
+    public function getCountryInfo(string $country = null, string $locale = null, int $maxRows = null, int $startRow = null): AddressCollection
```

Comment:
`AddressCollection` requires a collection of `Location`s and `CountryInfo` is not a `Location`. Can you advise how should this be handled?